### ISSUE TITLE
@jdalton

### ISSUE THREAD

**Issue by jdalton** [jkup/rejected.us#10]
TL;DR;
I was rejected by Google, Mozilla, and Twitter before being hired by Microsoft. No ragrets.

I was rejected by Google and Mozilla in 2011 and then by Twitter in 2012. All were on-site interviews and everyone involved was super friendly. The Google interview was a mixup. They somehow thought I was in for a CSS/HTML design position. I got to see the Google campus and meet Hedger Wang, [very old school cool](https://web.archive.org/web/20050625032247/http://www.hedgerwow.com/), so it was awesome! The Mozilla interview fell through because I wasn't open to relocation, although I'm not 100% sure on that.

In 2012, shortly after I released Lodash, I interviewed with Facebook, Twitter, & Microsoft. I was rejected by Twitter because I was too into JavaScript. I think they were looking for someone willing to juggle a few different languages.

All in all I'm really happy with how things turned out. I've gotten to help Chakra become one of fastest JS engines out there, improve JS at its core through the TC39, foster open source at Microsoft, contribute to many JS libraries/frameworks, and improve the web :metal:!

**Comment by axemclion** [comment]
:+1:

**Comment by berkerpeksag** [comment]
> I was rejected by Twitter because I was too into JavaScript.

I got a similar response from a Python company because I was too involved with Python :)

**Comment by jkup** [comment]
Thanks for sharing your story! Huge fan of Lodash and Chakra :) :rocket:
